permits/services.py

```diff
@@ -1001,14 +1004,25 @@ def request_permit_request_validation(permit_request, departments, absolute_uri_
         models.PermitRequestValidation.objects.get_or_create(
             permit_request=permit_request, department=department
         )
-
-    users_to_notify = {
-        email
-        for department in departments
-        for email in department.group.user_set.values_list(
-            "permitauthor__user__email", flat=True
+    if False:
```

Comment:
For the moment, I've set up that switch here because I'm not quite sure that the new version does exactly the same, especially that the variable `groups__permitdepartment__in` is equivalent to the for loop with `department.group.user_set` in the previous form.

Reply:
Found the [culprit](https://github.com/yverdon/geocity/blob/f4844ff7daaf89e9a414fed4a24ba0c2486d42ae/permits/services.py#L983):

```
data["users_to_notify"] = [permit_request.author.user.email]
```

So it SHOULD be fine with that.

Reply:
Removed old code in commit [5336e2b3a1542aacebba243d5e8cf78124a7e2a7](https://github.com/yverdon/geocity/pull/382/commits/5336e2b3a1542aacebba243d5e8cf78124a7e2a7).